x/stakeibc/types/message_add_validator.go

```diff
@@ -59,13 +59,6 @@ func (msg *MsgAddValidator) ValidateBasic() error {
 	if msg.Commission > 100 {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "commission must be between 0 and 100")
 	}
-	if msg.Commission > 10 {
-		fmt.Sprintf("WARNING: commission is %d (greater than 10pct)", msg.Commission)
-	}
-	// weight validation
-	if msg.Weight < 0 {
```

Comment:
is this covered already since the variable type is unsigned int?

Reply:
yes but nice to have the checks explicitly for uints IMO in case we mindlessly change the types, I could see that happening with the developer not realizing that the type was acting as a validation of sorts. redundancy here is worth the safety, IMO. wdyt?

Reply:
The linter disallows the check, but it's possible we can add a rule so that the linter ignores this